lib/Detector.js

```diff
@@ -38,17 +70,17 @@ const ENABLE_FEATURE_COMMENT = `${PLUGIN_OPTION_COMMENT}enable`;
  */
 export default class Detector {
   /**
-   * @param {(keyof FEATURES & string)[]} featureList an array of feature slugs (see caniuse-db)
+   * @param {(FeatureKeys & string)[]} featureList an array of feature slugs (see caniuse-db)
    */
   constructor(featureList) {
-    /** @type {Partial<FEATURES>} */
-    this.features = {};
-    for (const feature of featureList) {
-      if (FEATURES[feature]) {
-        this.features[feature] = FEATURES[feature];
-      }
-    }
-    /** @type {(keyof FEATURES & string)[]} */
+    /** @type {[FeatureKeys, RuleCheck][]} */
```

Comment:
Have you considered a map instead of a tuple array?

Reply:
great point, that actually simplifies a few things.